composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "project",
     "license": "GPLv3",
     "require": {
-        "convertkit/convertkit-wordpress-libraries": "1.1.0"
+        "convertkit/convertkit-wordpress-libraries": "1.2.0.x-dev"
```

Comment:
Is this library ready to be used in production? If so, should we drop the x-dev?

Reply:
Yes, this is now updated as [1.2.0 of the library is approved and published](https://github.com/ConvertKit/convertkit-wordpress-libraries/releases/tag/1.2.0). Thanks!